tests/acceptance/core/testcase/ingressdns.go

```diff
@@ -0,0 +1,50 @@
+package testcase
+
+import (
+	"github.com/k3s-io/k3s/tests/acceptance/core/service/assert"
+	"github.com/k3s-io/k3s/tests/acceptance/shared/util"
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+)
+
+func TestIngress(g ginkgo.GinkgoTestingT, deployWorkload bool) {
+	if deployWorkload {
+		_, err := util.ManageWorkload("create", "ingress.yaml", *util.Arch)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred(),
+			"Ingress manifest not deployed")
+	}
+
+	err := assert.ValidateOnHost(util.GetIngressRunning+util.KubeConfigFile, util.RunningAssert)
+	if err != nil {
+		ginkgo.GinkgoT().Logf("Error: %v", err)
```

Comment:
I'm confused why we are logging an error here rather than failing if it does exist.